core/src/main/targetconfigs/vz/application.yaml

```diff
@@ -1,6 +1,6 @@
 # Copyright (c) 2020, 2021, Oracle and/or its affiliates.
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
-​
+
```

Comment:
this doesn't show in review, but removing `<200b>` control character on this line.